presto-docs/src/main/sphinx/release/release-0.100.rst

```diff
@@ -22,3 +23,4 @@ General Changes
 * Rename config property ``task.shard.max-threads`` to ``task.max-worker-threads``.
   This property sets the number of threads used to concurrently process splits.
   The old property name is deprecated and will be removed in a future release.
+* Add deadlock detection to queue manager
```

Comment:
I would say "Improve query queue config validation" instead. The deadlocks are theoretic, rather than actual deadlocks which is think is what people normally think of as "deadlock detection"